RMSSD Function.R

```diff
@@ -0,0 +1,185 @@
+#RMSSD Function
+rm(list=ls())
+
+dataset <- `2019.7.10_RR_p36t0`
+
+#HR data may be outputed from equipment as RR values or ms/time values
+#let the individual specify what type of data they have
+
+data <- as.numeric(dataset[,2])
+x <- data.frame(data)
+
+
+
+
+#RMSSD FUNCTION
+#most common measure of heart rate variability 
+myRMSSD <- function(x, input, segment, max, min) {  
+
+  if (input == "RR" && segment == FALSE) {
+    N <- length(data)
+    N <- N-1
+    vector <- x^2
+    new <- sum(vector)/N
+    rmssd1 <- sqrt(new)
+    return(rmssd1)
+    
+  } else if (input == "MS" && segment == FALSE) {
+    RR <- diff(x)
+    RR <- as.numeric(RR)
+    N <- length(RR)
+    N <- N-1
+    vector <- RR^2
+    new <- sum(vector)/N
+    rmssd1 <- sqrt(new)
+    return(rmssd1)
+    
+  } else if (input == "RR" && segment == TRUE) {
+    N <- length(x)
+    N <- N-1
+    
+    #turning RR values back into milliseconds
+    ###
+    
+    
+    ########## Help please! @reviewer :)    ##########
+    #I am trying to make *Vector 2* a sum of all the previous elements in *Column 1*
+    #For example, if Column 1 is <- 12, 15, 10, 10, 26, 10
+        #then vector 2 should be <- 12, 27, 37, 47, 73, 83
+    #I feel like there is a simple answer but I can't seem to get it to work
+    #thanks!! 
+    
+    #this doesn't work:
+    sumVector <- 0
+    seq1 <- nrow(x)
+    seq2 <- (1:seq1)
+    
+    for (value in 1:length(seq2)) {
+      newsum <- sum(x[1:value, ])
+      sumVector <- append(sumVector, newsum)
+      return(sumVector)
+    }
+    #######################################
+    
+    
+    #convert millisecond to minutes so that the user can 
+    #input minutes as time constraints
+    #will adjust once above part is complete
+    vector1 <- x
+    vector2 <- vector1/1000
```

Comment:
Place spaces around all infix operators (this includes / ) 
--> vector2 <- vector1 / 1000